src/components/TextInput/index.jsx

```diff
@@ -1,7 +1,46 @@
-import React from "react";
+import React, { useState } from "react";
+import styles from "./styles/styles.module.scss";
 
-const TextInput = () => {
-  return <p>text input</p>;
+const TextInput = ({ label, minlength, isRequired = true }) => {
+  // let [inputStyle, setInputStyle] = useState(styles.input);
+  let [errorText, setErrorText] = useState("");
+
+  const handleInputValue = (event) => {
+    if (event.target.value.length === 0) {
```

Comment:
Put this is a variable and use that instead. The reason is that accessing object's properties has some processing costs + it's used twice.
```js
const valueLength = event.target.value.length;
```

Reply:
done